spec/unit/blockscore/collection/member_spec.rb

```diff
@@ -1,7 +1,32 @@
 module BlockScore
   class Collection
     RSpec.describe Member do
-      describe '.save' do
+      describe '#save' do
+        let(:question_set) { QuestionSet.new }
+        subject(:member) { described_class.new(person, question_set) }
+
+        context 'unsaved person' do
+          let(:person) { BlockScore::Person.new(attributes_for(:person)) }
+
+          it 'saves the person as part of member save' do
+            expect(person).to_not be_persisted
+            expect(member.save).to be true
+            expect(person).to be_persisted
+          end
+        end
+
+        context 'previously saved person' do
+          let(:person) { BlockScore::Person.create(attributes_for(:person)) }
+
+          it 'contains the proper fields' do
```

Comment:
Thx. There is a lot lacking in this - hence why this is at best a very thin first attempt. It was better than `describe '.save' do ; end` but not by much. It was enough to discover loads of issues with `#save`.